packages/components/src/components/post-avatar/post-avatar.tsx

```diff
@@ -163,6 +213,12 @@ export class PostAvatar {
     window?.sessionStorage?.removeItem(key);
   }
 
+  private async refreshAvatar() {
+    await this.removeStorageItem(this.userid);
+    await this.removeStorageItem(this.email);
+    this.getAvatar();
+  }
```

Comment:
I'm not sure about this function: `getAvatar` includes logic to retrieve a cached image, but the cached image is being removed right before it's called. This seems to defeat the purpose of the cache.